### ISSUE TITLE
Color contrast test incorrect on some font sizes

### ISSUE THREAD

**Issue by timmhayes** [Khan/tota11y#116]
The color contrast tool does not work correctly on fonts that are sized with decimals. I've seen this fail in Chrome with an element that had a size of 10pt, which was rendered at 13.3333px and showed up as a large font that only needs to have a 3.0 contrast ratio.

See a sample of this here:
http://output.jsbin.com/vutotup

I looked through your code and found the issue, but it is a problem in the `axs.utils.isLargeFont` function of the GoogleChome accessibility-developer-tools. So opened this issue for you:
https://github.com/GoogleChrome/accessibility-developer-tools/issues/345

and created this pull request:
https://github.com/GoogleChrome/accessibility-developer-tools/pull/346

I hope it works out for you, since I really like the tool. Thanks!

**Comment by mlp73** [comment]
Same issue here. The extension is throwing "low contrast ratio" fail on fonts which are compiled over 24px and pass the accepted contrast ratio.

**Comment by timmhayes** [comment]
My [pull request ](https://github.com/GoogleChrome/accessibility-developer-tools/pull/346) on Google's accessibility-developer-tools has sat unmerged for the last 9 months, so I simply made the change in my local version of tota11y.  You can see the unminified changes [in Google's repository](https://github.com/GoogleChrome/accessibility-developer-tools/pull/346/files). I hope that helps.
